docs/docs/30-administration/22-backends/40-kubernetes.md

```diff
@@ -308,3 +350,40 @@ Determines if containers must be required to run as non-root users.
 > Default: empty
 
 Secret names to pull images from private repositories. See, how to [Pull an Image from a Private Registry](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/).
+
+## Metrics
+
```

Comment:
And this. Move in deployment instruction or in helm chart repository.
https://woodpecker-ci.org/docs/administration/deployment-methods/kubernetes